Modules/hpe3par_snapshot.py

```diff
@@ -526,9 +526,10 @@ def create_schedule(
 
     expiration_hours = convert_to_hours(expiration_time, expiration_unit)
     retention_hours = convert_to_hours(retention_time, retention_unit)
-    if expiration_hours <= retention_hours:
-        return (False, False, "Expiration time must be \
-greater than retention time", {})
+    if not(not expiration_hours or not retention_hours):
```

Comment:
Can we put this as a single if condition?
if expiration_hours and retention_hours and expiration_hours <= retention_hours:
    return (False, False, "Expiration time must be \
greater than retention time for non zero values", {})